pkg/apps/blackduck/latest/containers/uploadcache.go

```diff
@@ -37,7 +37,7 @@ func (c *Creater) GetUploadCacheDeployment(imageName string) (*components.Replic
 			MinCPU: "", MaxCPU: ""},
 		EnvConfigs: []*horizonapi.EnvConfig{
 			c.getHubConfigEnv(),
-			// {NameOrPrefix: "SEAL_KEY", Type: horizonapi.EnvFromSecret, KeyOrVal: "SEAL_KEY", FromName: "upload-cache"},
+			// &horizonapi.EnvConfig{NameOrPrefix: "BLACKDUCK_UPLOAD_CACHE_HOST", Type: horizonapi.EnvVal, KeyOrVal: "0.0.0.0"},
```

Comment:
Not sure I understand the diff here; if the comment needs to be uncommented later on once the hub is fixed, maybe add a TODO here as well.

Reply:
removed the commented part